app/models/container_build.rb

```diff
@@ -4,10 +4,10 @@ class ContainerBuild < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   belongs_to :container_project
 
-  has_many :labels, -> { where(:section => "labels") },
+  has_many :labels, -> { where(:section => "labels") }, # rubocop:disable Rails/HasManyOrHasOneDependent
            :class_name => "CustomAttribute",
            :as         => :resource,
-           :dependent  => :destroy
```

Comment:
Just asking, is nullify the default if you don't specify it?  I think so but want to make sure that's the intention:  https://guides.rubyonrails.org/association_basics.html

Reply:
no. not specifying means it ignores it
`CustomAttributeMixin#has_many :custom_attribututes, :dependent => :destroy` will do the destroying. Pretty sure this is included in all models but one.

Reply:
> all models but one

Just to follow up: I checked all models.

`GuestDevice` does not have that mixin, but also we are not modifying custom attributes for that model, so that point is moot.

I added the detailed list in another thread.